test/azure-cli/scenarios.json

```diff
@@ -9,6 +9,16 @@
       }
     }
   },
+  "install_bicep_noble": {
```

Comment:
Can we remove this test? We already have workflows in place. Thanks!

Reply:
removed